src/components/subscriptions/AddUsersButton.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import AddUsersModal from '../../containers/AddUsersModal';
+import ActionButtonWithModal from '../ActionButtonWithModal';
+
+const AddUsersButton = ({
+  onSuccess,
+  onClose,
+}) => (
+  <ActionButtonWithModal
+    buttonLabel="Add Users"
+    buttonClassName="add-btn btn btn-primary float-right"
+    renderModal={({ closeModal }) => (
+      <AddUsersModal
+        couponId="1"
```

Comment:
What is `couponId`? Don't think it's relevant for subscription licenses 😄 Is this meant to be the `SubscriptionPlan` uuid?